external/LibraryThemeStyles/src/Service/ControllerFactory.php

```
@@ -11,15 +11,10 @@
  */
 class ControllerFactory implements FactoryInterface
 {
-    /**
-     * Create an AdminController instance configured with the module configuration service.
-     *
-     * @return AdminController The constructed AdminController instance configured with the module configuration service.
-     */
     public function __invoke(ContainerInterface $container, $requestedName, array $options = null): AdminController
```

PHPMD (2.15.0)

Avoid unused parameters such as '$requestedName'. (undefined)

(UnusedFormalParameter)